CHANGELOG.md

```diff
@@ -76,6 +76,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - server: raise error on startup when `--unauthorized-role` is ignored (#4736)
 - server: fix mishandling of GeoJSON inputs in subscriptions (fix #3239)
 - server: fix importing of allow list query from metadata (fix #4687)
+- server: add new `--stale-timeout` and `HASURA_GRAPHQL_PG_STALE_TIMEOUT` connection pool options, to work around space leak caused by libpq buffers (#5087)
```

Comment:
Do we still need "to work around space leak caused by libpq buffers" as I presume that is coming in a different PR ?

Reply:
This PR contains the libpq fix for buffer shrinking. I think it's worthwhile to keep the new "lifetime" feature (formerly workaround for the buffer issue): if there's a reason that it is in pgbouncer I suppose it might be useful for some reason we maybe haven't thought about to hasura users. Editing the changelog though